.github/workflows/test.yml

```diff
@@ -224,6 +224,7 @@ jobs:
         with:
           repo: gotestyourself/gotestsum
           tag: v1.8.1
+          cache: enable
```

Comment:
Actually something is off here, why are we using aaronfriel/action-install-gh-release@v1.4.0-beta.4 and not jaxxstorm/action-install-gh-release@v1.7.1

@AaronFriel  would you know ?

Reply:
Discussed that this change was upstreamed, so we do not need to reference the fork anymore.

Reply:
needed to grab .zip files for gotestsum on Windows, IIRC. it's upstreamed now.